src/vs/workbench/contrib/welcome/page/browser/welcomePage.ts

```diff
@@ -29,7 +29,7 @@ import { ILifecycleService, StartupKind } from 'vs/platform/lifecycle/common/lif
 import { Disposable } from 'vs/base/common/lifecycle';
```

Comment:
With this many changes I think we should just make our own copy of this and redirect the contribution to use that instead. Thoughts @anthonydresser?